qr_generator/lib/model/seed_model.dart

```diff
@@ -0,0 +1,30 @@
+class SeedModel {
```

Comment:
**question:** could this class be immutable?

Reply:
Yes, it could. I already changed it.